connect/mirror/src/test/java/org/apache/kafka/connect/mirror/MirrorConnectorsIntegrationTest.java

```diff
@@ -315,9 +319,34 @@ public void testReplication() throws InterruptedException {
             backup.kafka().consume(NUM_RECORDS_PRODUCED, 2 * RECORD_TRANSFER_DURATION_MS, "primary.test-topic-2").count());
     }
 
+    private void waitForConsumerGroupOffsetSync(Consumer<byte[], byte[]> consumer, List<String> topics)
+            throws InterruptedException {
+        Admin backupClient = backup.kafka().createAdminClient();
+        List<TopicPartition> tps = new ArrayList<>(NUM_PARTITIONS * topics.size());
+        IntStream.range(0, NUM_PARTITIONS).forEach(
+            partitionInd -> {
```

Comment:
could we have a more intuitive variable name for `partitionInd`? e.g. `partitionId` or `partitionIndex`?

Reply:
I'll rename to `partitionIndex`. Thanks.